dev/benchmarks/macrobenchmarks/lib/src/web/recorder.dart

```diff
@@ -321,6 +323,18 @@ abstract class WidgetBuildRecorder extends Recorder
   /// consider using [WidgetRecorder].
   Widget createWidget();
 
+  /// Called once before all runs of this benchmark recorder.
+  ///
+  /// This is useful for doing one-time setup work that's needed for the
+  /// benchmark.
+  void setUpAll() {}
```

Comment:
not needed if you don't use.

Reply:
it is being overriden in `bench_text_layout.dart`.